tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "compilerOptions": {
+    "noEmit": true,
+    "noImplicitAny": true,
+    "target": "ES2015"
+  },
```

Comment:
I know this is already merged but I recommend to set `strict: true`. just mentioning so that it can be considered in the future. If it was intentional to not set it then feel free to ignore the comment